sansa-ml-spark/pom.xml

```diff
@@ -61,6 +61,12 @@
 			<artifactId>spark-mllib_${scala.binary.version}</artifactId>
 			<version>${spark.version}</version>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/graphframes/graphframes -->
+        <dependency>
+            <groupId>graphframes</groupId>
+            <artifactId>graphframes</artifactId>
+            <version>0.7.0-spark2.4-s_2.11</version>
```

Comment:
Great! Thanks for adding the `graphframes` dependency. The project seems to build now. Consider using the latest version i.e. `0.8.0` and also format it :) -- i.e. align with other dependency lists.

Reply:
You have to add a Maven repo, nobody wants to add local Jars in the project classpath manually nowadays.

It is also mentioned on the [GraphFrames Maven artifact page](https://mvnrepository.com/artifact/graphframes/graphframes/0.8.0-spark2.4-s_2.11):

> Note: this artifact it located at SparkPackages repository (https://dl.bintray.com/spark-packages/maven/) 

That means, we have to add

```
<repository>
			<id>SparkPackages</id>
			<name>Repo for Spark packages</name>
			<url>https://dl.bintray.com/spark-packages/maven/</url> <!-- the https you've been looking for -->
			<snapshots>
				<enabled>false</enabled> <!-- or set to true if desired, default is false -->
			</snapshots>
		</repository>
```